app/controllers/users_controller.rb

```diff
@@ -4,7 +4,12 @@ def index
     @users = User.all
   end
 
-  def show; end
+  def show
+    @my_reservations = current_user.borrowed_books
```

Comment:
I am a little bit confused here, are we interested in reservations or borrowings? I think we need to make a distinction between books that a user has reserved, and ones that they have actually borrowed.

I am thinking that instead of using `has_many` association, we could add two scopes in the `Book` model:
```ruby
  scope :reserved_by, ->(user) {
    where(borrower: user, status: 'reserved')
  }
```

```ruby
  scope :borrowed_by, ->(user) {
    where(borrower: user, status: 'borrowed')
  }
```

Reply:
I'm not sure what we have finally agreed on last week, but I guess this functionality now exists thanks to `enum`:
`user.books.reserved` and `user.books.borrowed`